src/App.js

```diff
@@ -1,13 +1,25 @@
+import useAxios from "./customizedhooks/useAxios"
+
 function App() {
+  const [status, loader, data] = useAxios("/ping")
+
   return (
     <>
-    <header className="px-8 py-4 text-xl font-semibold shadow-sm">
-      <h1>Coindashboard</h1>
-    </header>
-    <div className="container mx-auto text-center md:w-2/3">
-      <h1 className="text-6xl font-bold text-indigo-600 md:text-7xl mt-44 rotate-6 ">Hello ❤️ Capgemini!</h1>
-    </div>
-   
+      <header className="px-8 py-4 text-xl font-semibold shadow-sm">
+        <h1>Coindashboard</h1>
+      </header>
+      <div className="container mx-auto text-center md:w-2/3">
+        <h1 className="text-6xl font-bold text-indigo-600 md:text-7xl mt-44 rotate-6 ">Hello ❤️ Capgemini!</h1>
+      </div>
+      {/* ADDED */}
+      {!loader ? status !== 200 ?
+        <h1>Server error message : {status.message}</h1>
+        :
+        <h1>Server response DATA: {data.gecko_says}</h1>
+        :
+        <h1>LOADING DATA.....</h1>
```

Comment:
if not Loading Data (Downloading Data from the server finished) and if The status of HTTP response is not 200 which mean we got an error (404 for example ) we display the ERROR MESSAGE, if The status of HTTP response is 200 then we display The data ( in this case is just a ping so the response contain just gecko_says) in Otherwise if The Loading is True we display the Loader. I hope it's more clear. ( you can check the API website to see what Kind of Data you can obtain just by changing the Path in the URL in our case The URL passed useAxios.